parser_library/src/parsing/parser_impl.cpp

```diff
@@ -537,11 +541,12 @@ void parser_impl::process_lookahead()
             parse_lookahead_operands(std::move(*look_lab_instr->op_text), look_lab_instr->op_range);
     }
 }
-void parser_impl::transform_imm_reg_operands(semantics::collector& col, id_index instruction)
+void parser_impl::transform_imm_reg_operands(semantics::operand_list& op_list, id_index instruction)
 {
+    if (instruction->empty())
```

Comment:
when does this happen?

Reply:
There is a test "parse_bad_model" where it occurs